docker/single-process/scripts/init

```diff
@@ -52,20 +52,22 @@ esac
 
 sudo -u huginn -H -E bundle install --without test development --path vendor/bundle
 
+export RAILS_ENV=${RAILS_ENV}
+
 if [[ -z "${DO_NOT_CREATE_DATABASE}" && -z $1 ]]; then
-  sudo -u huginn -H -E bundle exec rake db:create RAILS_ENV=${RAILS_ENV}
```

Comment:
removing `RAILS_ENV` as an argument allows downstream scripts to quickly check the presence of `"$@"` to execute default or custom commands